api/apps/api/test/project-scenarios.e2e-spec.ts

```diff
@@ -223,13 +261,39 @@ async function getFixtures() {
       });
     },
 
+    GivenPreviousScenario: async (name: string, projectScenarioId: number) => {
+      const scenario = await scenariosRepo.save({
+        name,
+        type: ScenarioType.marxan,
+        projectId,
+      });
+      await scenariosRepo.save({
+        id: scenario.id,
+        projectScenarioId,
+      });
+    },
+
     WhenCreatingAScenarioWithIncompleteData: async () =>
       await request(app.getHttpServer())
         .post('/api/v1/scenarios')
         .set('Authorization', `Bearer ${ownerToken}`)
         .send(E2E_CONFIG.scenarios.invalid.missingRequiredFields()),
 
-    WhenCreatingAScenarioWithMinimumRequiredDataAsOwner: async () => {
+    WhenCreatingAScenarioWithMinimumRequiredDataAsOwner: async (
+      grabId = true,
+    ) => {
+      const response = await request(app.getHttpServer())
+        .post('/api/v1/scenarios')
+        .set('Authorization', `Bearer ${ownerToken}`)
+        .send(minimalCreateScenarioDTO);
+
+      if (grabId) {
+        scenarioId = response.body.data.id;
+      }
+      return response;
+    },
+
+    WhenCreatingAScenarioWithOverMaxProjectScenarioId: async () => {
```

Comment:
is this accidentally incomplete? there's no explicit handling of project_scenario_id, and from what I can see the function is not even used in any test

Reply:
Not used, forgot to remove after reconsidering the approach for that test